Silicon/NXP/iMXPlatformPkg/Drivers/GopNullDxe/GopNullDxe.c

```diff
@@ -0,0 +1,271 @@
+/** @file
+*
+*  Copyright (c) 2019 Microsoft Corporation. All rights reserved.
+*  Copyright 2018 NXP
+*
+*  This program and the accompanying materials
+*  are licensed and made available under the terms and conditions of the BSD License
+*  which accompanies this distribution.  The full text of the license may be found at
+*  http://opensource.org/licenses/bsd-license.php
+*
+*  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+*
+**/
+
+#include "GopNullDxe.h"

```

Comment:
Shouldn't a Gop 'Null' Dxe not actually do anything?

Reply:
True. I wanted to highlight that this Gop driver does not actually initialize HW or query the underlying platform. It statically sets a 1920x1080 framebuffer and exposes the minimum set of GOP modes to the rest of EDK2/OS.

I am open to other naming suggestions 😃